ddl/ddl_api.go

```diff
@@ -3277,6 +3277,9 @@ func (d *ddl) RebaseAutoID(ctx sessionctx.Context, ident ast.Ident, newBase int6
 			return err
 		}
 		if newBase != newBaseTemp {
+			if newBaseTemp < 0 {
```

Comment:
it seems `adjustNewBaseToNextGlobalID` will return a int64, what if the auto_increment is unsigned?